CoEpi/ui/onboarding/Strings.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+struct Strings {
+    
+    struct onboarding {
```

Comment:
We should use `SwiftGen` to generate these keys. What do you think about removing localization from this PR and we implement it properly in a future version (considering also that the bold functionality isn't localizable)?

Reply:
Done. There is now a struct containing the new markup strings, just to keep it out of the view controller. Localization has been removed.